README.md

```diff
@@ -43,6 +43,59 @@ Please choose one of the profile when building the war file
 * public - core + API + public website (deprecated)
 
 You could find specific instructions in curate or public repo,
-         
+
+## Run with Docker containers
+
+**OncoKB Core**
+1. OncoKB Core requires a MySQL server and the `oncokb-core` and `oncokb-transcript` databases imported. This step must be completed before contining the installation process. Reach out to contact@oncokb.org to get access to the data dump. 
+
+<br>
+
+OncoKB Core also uses [Genome Nexus](https://github.com/genome-nexus/genome-nexus) to retrieve alteration annotations. By default, the API requests are sent to www.genomenexus.org for GRCh37 and grch38.genomenexus.org for GRCh38. However, you can choose to use a local version of Genome Nexus by following the instructions for *Option A*, otherwise follow instructions for *Option B*.
```

Comment:
```suggestion
OncoKB also uses [Genome Nexus](https://github.com/genome-nexus/genome-nexus) to annotate genomic change to protein change using OncoKB picked transcripts. By default, the API requests are sent to www.genomenexus.org for GRCh37 and grch38.genomenexus.org for GRCh38. However, you can choose to use a local version of Genome Nexus by following the instructions for *Option A*, otherwise follow instructions for *Option B*.
```